src/transformers/models/vit/modeling_vit.py

```diff
@@ -422,10 +435,6 @@ def _init_weights(self, module):
             module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
             if module.bias is not None:
                 module.bias.data.zero_()
-        elif isinstance(module, nn.Embedding):
-            module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
-            if module.padding_idx is not None:
-                module.weight.data[module.padding_idx].zero_()
```

Comment:
Why is this removed?

Reply:
Closing this since no `nn.Embedding` layer is used in modeling_vit.py